onnxscript/function_libs/torch_aten/graph_building_test.py

```diff
@@ -17,23 +17,22 @@
 @unittest.skipIf(version_utils.torch_older_than("2.0"), "torchscript in 1.13 not supported")
 class TestTorchScriptTracingEvaluator(unittest.TestCase):
     def setUp(self):
+        # FIXME: Currently this must match with the import line
+        # `from onnxscript import opset17 as op`, which restricts opset to be 17 in these
```

Comment:
The model opset can actually be different from the local functions' opsets I think. Do you know if that's true?

Reply:
Yea it could be different under constraint.
https://github.com/onnx/onnx/blob/4b2d50334914621835cc1e8dadd4fe82b6b9876c/onnx/onnx.in.proto#L824-L828
```
  // The operator sets imported by FunctionProto should be compatible with the ones
  // imported by ModelProto. Example, if same operator set say 'A' is imported by FunctionProto
  // and ModelProto then versions for the operator set may be different but,
  // the operator schema returned for op_type, domain, version combination
  // for both the versions should be same.
```